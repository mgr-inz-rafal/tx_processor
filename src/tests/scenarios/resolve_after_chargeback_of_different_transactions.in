type,client,tx,amount
deposit,1,1,100.0
deposit,1,2,200.0
dispute,1,1,
dispute,1,2,
chargeback,1,1,
resolve,1,2,
