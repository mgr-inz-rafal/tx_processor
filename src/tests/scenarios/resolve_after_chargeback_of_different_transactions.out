client,available,held,total,locked
1,0,200,200,true